### PR TITLE
[PATCH ArtworkDetails] Include follow partner profile button

### PR DESCRIPTION
**Pull request by oxaudo**
@damassi  also helped me in refactoring default exports into named exports on both FollowArtist, followProfile and followGene buttons.

### CODE REVIEW

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit ad556a123e42182de7571539bd91dfdfe3574e91

https://deploy-preview-1361--artsy-reaction.netlify.com

File: src/Apps/Artist/Components/ArtistHeader.tsx [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { ArtistHeader_artist } from "__generated__/ArtistHeader_artist.graphql"
 import { track, Track } from "Artsy/Analytics"
 import * as Schema from "Artsy/Analytics/Schema"
 import { ContextConsumer, Mediator } from "Artsy/SystemContext"
-import FollowArtistButton from "Components/FollowButton/FollowArtistButton"
+import { FollowArtistButtonFragmentContainer as FollowArtistButton } from "Components/FollowButton/FollowArtistButton"
```
Comment (damassi) [inline src/Apps/Artist/Components/ArtistHeader.tsx]: 👍

**Comment by damassi** [comment]
LGTM 👍 Thanks for cleaning up those other legacy callsites

File: src/Apps/Artwork/Components/ArtworkDetails/ArtworkDetailsAboutTheWorkFromPartner.tsx [hunk 2]
```diff
@@ -74,7 +74,7 @@ export class ArtworkDetailsAboutTheWorkFromPartner extends React.Component<
                                       kind: "profile",
                                       action: "follow",
                                       objectId:
-                                        partner.profile && partner.profile.__id,
+                                        partner.profile && partner.profile.id,
```
Comment (mzikherman) [inline src/Apps/Artwork/Components/ArtworkDetails/ArtworkDetailsAboutTheWorkFromPartner.tsx]: ah right, god catch!

**Pull request merged**